src/crimson/os/seastore/transaction_manager.h

```diff
@@ -34,6 +34,20 @@
 namespace crimson::os::seastore {
 class Journal;
 
+struct tm_make_config_t {
+  bool detailed = true;
+  journal_type_t j_type = journal_type_t::SEGMENT_JOURNAL;
+  placement_hint_t default_placement_hint =
+    placement_hint_t::HOT;
+  static tm_make_config_t get_default() {
+    return tm_make_config_t {
+      true,
```

Comment:
This should only be set to true for tests.  I'll submit a PR shortly to fix this https://tracker.ceph.com/issues/55706